corehq/apps/users/forms.py

```diff
@@ -413,9 +414,14 @@ def __init__(self, project, user_id, **kwargs):
         )
 
     def clean_new_password1(self):
```

Comment:
Perhaps you misunderstand, if this is getting overridden then it won't get called at all unless the function in `EncodedPasswordChangeForm` calls it explicitly with `super`.

As it stands right now this function will never get called.